{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"ues"},{"kind":"text","text":": "},{"kind":"typeIdentifier","identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/x86_exception_state-swift.struct","preciseIdentifier":"c:@S@x86_exception_state","text":"x86_exception_state"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"__Unnamed_union_ues"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/nio\/x86_exception_state-swift.struct\/ues"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/org.swift.docc\/documentation\/NIO\/x86_exception_state-swift.struct\/ues","interfaceLanguage":"swift"},"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"ues"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"x86_exception_state","preciseIdentifier":"c:@S@x86_exception_state"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"__Unnamed_union_ues"}],"title":"ues","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"c:@S@x86_exception_state@FI@ues","modules":[{"name":"NIO"}]},"hierarchy":{"paths":[["doc:\/\/org.swift.docc\/documentation\/NIO","doc:\/\/org.swift.docc\/documentation\/NIO\/x86_exception_state-swift.struct"]]},"references":{"doc://org.swift.docc/documentation/NIO/x86_exception_state-swift.struct":{"role":"symbol","title":"x86_exception_state","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"x86_exception_state"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/x86_exception_state-swift.struct","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"x86_exception_state"}],"url":"\/documentation\/nio\/x86_exception_state-swift.struct"},"doc://org.swift.docc/documentation/NIO":{"role":"collection","title":"NIO","abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO","kind":"symbol","type":"topic","url":"\/documentation\/nio"},"doc://org.swift.docc/documentation/NIO/x86_exception_state-swift.struct/ues":{"role":"symbol","title":"ues","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"ues"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"x86_exception_state","preciseIdentifier":"c:@S@x86_exception_state"},{"kind":"text","text":"."},{"kind":"typeIdentifier","text":"__Unnamed_union_ues"}],"abstract":[],"identifier":"doc:\/\/org.swift.docc\/documentation\/NIO\/x86_exception_state-swift.struct\/ues","kind":"symbol","type":"topic","url":"\/documentation\/nio\/x86_exception_state-swift.struct\/ues"}}}